client/src/main.rs

```diff
@@ -85,6 +85,8 @@ async fn main_inner() -> Result<(), Error> {
         use nimiq::extras::rpc_server::initialize_rpc_server;
         let rpc_server = initialize_rpc_server(&client, rpc_config, client.wallet_store())
             .expect("Failed to initialize RPC server");
+        // false positive: https://github.com/rust-lang/rust-clippy/issues/10482
+        #[allow(clippy::redundant_async_block)]
```

Comment:
The CI is reporting this as an unknown lint :thinking:

Reply:
Should I do it like the matrix-rust-sdk commit (i.e. `#[allow(unknown_lints, clippy::redundant_async_block)]`) or just drop the `#[allow]` until the CI gets the lint?

https://github.com/matrix-org/matrix-rust-sdk/commit/8d83a996a69ffcfca786dbc343dc9b862a9965df#diff-f4219660dd4687809f8c9f0a5f4f17b5e782ea7e28f5b6e40cccbb67c38e1308R380

Reply:
Mmm, I'd vote for removing it until the CI gets it because it's a false positive that is probably not in stable yet?